src/Sylius/Bundle/AdminBundle/Resources/config/routing/ajax/product_variant.yml

```diff
@@ -30,7 +30,5 @@ sylius_admin_ajax_product_variants_by_code:
             serialization_groups: [Autocomplete]
             permission: true
             repository:
-                method: findByCodeAndProductCode
-                arguments:
-                    code: $code
-                    productCode: $productCode
+                method: findByCodesAndProductCode
+                arguments: [$code, $productCode]
```

Comment:
$code -> $codes

Reply:
It's not that easy, because this data is collected in js file, where we have `choice_value` as `code` and I guess that I shouldn't change that.